packages/agoric-cli/src/lib/wallet.js

```diff
@@ -28,13 +29,28 @@ const emptyCurrentRecord = {
 export const getCurrent = async (addr, { readLatestHead }) => {
   // Partial because older writes may not have had all properties
   // NB: assumes changes are only additions
-  const current =
+  let current =
     /** @type {Partial<import('@agoric/smart-wallet/src/smartWallet').CurrentWalletRecord> | undefined} */ (
       await readLatestHead(`published.wallet.${addr}.current`)
     );
   if (current === undefined) {
     throw new Error(`undefined current node for ${addr}`);
   }
+
+  // Repair a type misunderstanding seen in the wild
```

Comment:
We can link to https://github.com/Agoric/agoric-sdk/pull/7139 to be more clear

Reply:
Aha! Makes it more clear to me to.

Done, thanks.